### PR TITLE
documented our sprint planning (and pre-planning) process

### PR DESCRIPTION
**Pull request by chrispine**

### CODE REVIEW

File: handbook/engineering/campaigns/index.md [hunk 1]
```diff
@@ -47,11 +47,21 @@ When I create a campaign to make large-scale code changes I want to _focus on th
...
+* Before the iteration begins, we do pre-planning to make the most of our the planning meeting:
+  * EM creates a [tracking issue](https://about.sourcegraph.com/handbook/engineering/tracking_issues) for the iteration which will hold our goals and plans, in addition to tracking the issues we intend to address.
+  * Everyone on the team looks through [our backlog](https://github.com/sourcegraph/sourcegraph/issues?q=is%3Aopen+is%3Aissue+label%3Ateam%2Fcampaigns+milestone%3ABacklog) for any issues they think we should consider for the next sprint, and set the corresponding Campaigns Sprint milestone.
+  * Everyone on the team looks through the current tracking issue for any issues they do not expect to finish by the end of the sprint (EOD Tuesday), and set the next sprint's milestone so they will show up on  the new tracking issue.
+  * Everyone on the team reviews the roadmap doc and comments their thoughts. Replace estimate placeholders with your actual estimates (and a best-guess range is fine: "3-5d"). For any items without linked Github issues, please create those issues and link them.
```
Comment (chrispine) [inline handbook/engineering/campaigns/index.md:54]:
```suggestion
  * Engineers add estimates to issues in the new tracking issue if they are missing.
  * Everyone on the team reviews the roadmap doc and comments their thoughts. Replace estimate placeholders with your actual estimates (and a best-guess range is fine: "3-5d"). For any items without linked Github issues, please create those issues and link them.
```

File: handbook/engineering/campaigns/index.md [hunk 2]
```diff
@@ -47,11 +47,22 @@ When I create a campaign to make large-scale code changes I want to _focus on th
...
-* We then have our planning meeting to determine our common goals for the iteration. We create a [tracking issue](https://about.sourcegraph.com/handbook/engineering/tracking_issues) for the iteration which will hold our goals and plans, in addition to tracking the issues we intend to address.
-* Each Monday, the team meets with stakeholders to stay in sync, and to set our personal goals for the week. We record these goals, and the previous week's progress, as status update comments on the tracking issue. The engineering manager then compiles these into a summary which is sent to the team and stakeholders.
+* Before the iteration begins, we do pre-planning to make the most of our the planning meeting:
+  * EM creates a [tracking issue](https://about.sourcegraph.com/handbook/engineering/tracking_issues) for the iteration which will hold our goals and plans, in addition to tracking the issues we intend to address.
```
Comment (eseliger) [inline handbook/engineering/campaigns/index.md:51]: And perhaps the milestone for N+1 and a label `planned/CampaignsSprintN`?

**Review by LawnGnome — APPROVED** [review]
I share Erik's question, but otherwise I think this lines up with what we've been trending towards the last few sprints. :+1:

File: handbook/engineering/campaigns/index.md [hunk 3]
```diff
@@ -47,11 +47,22 @@ When I create a campaign to make large-scale code changes I want to _focus on th
...
+  * Everyone on the team looks through [our backlog](https://github.com/sourcegraph/sourcegraph/issues?q=is%3Aopen+is%3Aissue+label%3Ateam%2Fcampaigns+milestone%3ABacklog) for any issues they think we should consider for the next sprint, and set the corresponding Campaigns Sprint milestone.
```
[the remark this answers is not in the document]
Comment (chrispine) [reply handbook/engineering/campaigns/index.md:52]: I think it's fine not to do this until the planning meeting (though they are welcome to assign them to themselves if they want).

File: handbook/engineering/campaigns/index.md (hunk 2 above)
Comment (chrispine) [reply handbook/engineering/campaigns/index.md:51]: Good ideas!

Comment (chrispine) [inline handbook/engineering/campaigns/index.md:51]:
```suggestion
  * EM creates a [tracking issue](https://about.sourcegraph.com/handbook/engineering/tracking_issues) for the iteration which will hold our goals and plans, in addition to tracking the issues we intend to address. EM also creates the milestone for sprint N+1 and a `planned/CampaignsSprintN` label for things that didn't get finished in the previous sprint.
```

**Pull request merged**